src/agent/onefuzz/src/az_copy.rs

```diff
@@ -15,7 +15,9 @@ pub async fn sync(src: impl AsRef<OsStr>, dst: impl AsRef<OsStr>) -> Result<()>
         .stderr(Stdio::piped())
         .arg("sync")
         .arg(&src)
-        .arg(&dst);
+        .arg(&dst)
+        .arg("--delete_destination")
+        .arg(delete_dst.to_string());
```

Comment:
Does this work the same as passing `--delete-destination=$bool`? (i.e. with an `=` instead of space)?

Reply:
updated the logic to only add --delete-destination if delete_dst is true